scm/github.go

```diff
@@ -704,53 +705,83 @@ func (s *GithubSCM) CreateIssue(ctx context.Context, opt *CreateIssueOptions) (*
 		Assignees: opt.Assignees,
 	}
 
-	s.logger.Debugf("CreateRepository: creating issue %s on %s Repository", opt.Title, opt.Repository)
+	s.logger.Debugf("Creating issue %s on %s Repository", opt.Title, opt.Repository)
 	issue, _, err := s.client.Issues.Create(ctx, opt.Organization, opt.Repository, newIssue)
 	if err != nil {
 		return nil, ErrFailedSCM{
 			Method:   "CreateIssue",
-			Message:  fmt.Sprintf("failed to create Issue %s, make sure all the details are correct ", opt.Title),
+			Message:  fmt.Sprintf("failed to create issue %s, make sure all the details are correct ", opt.Title),
```

Comment:
Probably use `%q` also here. The last portion with `, make sure...` may not be so useful... If keep it, remove the last space after `correct`